config/locales/en/breadcrumbs.yml

```diff
@@ -102,6 +102,8 @@ en:
           edit_topic: :'course.forum.topics.edit_topic.header'
       leaderboards:
         index: :'course.leaderboard.title'
+      experience_points_records:
+        index: :'course.experience_points_records.index.title'
```

Comment:
Should I put the string here directly instead of using a reference, since `i18n-tasks` complains about `course.experience_points_records.index.title` being an unused key?